diff -urN metadata-extractor-2.3.1/src/com/drew/metadata/exif/NikonType2MakernoteDirectory.java metadata-extractor-2.3.1-patched/src/com/drew/metadata/exif/NikonType2MakernoteDirectory.java
--- metadata-extractor-2.3.1/src/com/drew/metadata/exif/NikonType2MakernoteDirectory.java	2005-10-16 15:59:00.000000000 +1100
+++ metadata-extractor-2.3.1-patched/src/com/drew/metadata/exif/NikonType2MakernoteDirectory.java	2012-02-21 04:51:24.000000000 +1100
@@ -149,7 +149,6 @@
 
     /**
      * Values observed
-     * - מ
      * - 0,1,c,0 (hex)
      */
     public static final int TAG_NIKON_TYPE2_UNKNOWN_2 = 0x000E;
@@ -295,7 +294,6 @@
     /**
      * Values observed
      * - 48,1,c,0 (hex) (D100)
-     * - @
      */
     public static final int TAG_NIKON_TYPE2_UNKNOWN_8 = 0x008B;
 
@@ -326,7 +324,6 @@
 
     /**
      * Values observed
-     * - 0100)
      * - 0103 (D70)
      * - 0100 (D1X)
      */
@@ -348,16 +345,11 @@
 
     /**
      * Values observed
-     * - 0100'~e3
-     * - 0103
      */
     public static final int TAG_NIKON_TYPE2_UNKNOWN_12 = 0x0097;
 
     /**
      * Values observed
-     * - 0100fht@7b,4x,D"Y
-     * - 01015
-     * - 0100w\cH+D$$h$מ5Q (D1X)
      * - 30,31,30,30,0,0,b,48,7c,7c,24,24,5,15,24,0,0,0,0,0 (hex) (D100)
      */
     public static final int TAG_NIKON_TYPE2_UNKNOWN_13 = 0x0098;
diff -urN metadata-extractor-2.3.1/src/com/drew/metadata/exif/test/ExifDescriptorTest.java metadata-extractor-2.3.1-patched/src/com/drew/metadata/exif/test/ExifDescriptorTest.java
--- metadata-extractor-2.3.1/src/com/drew/metadata/exif/test/ExifDescriptorTest.java	2005-10-16 15:59:00.000000000 +1100
+++ metadata-extractor-2.3.1-patched/src/com/drew/metadata/exif/test/ExifDescriptorTest.java	2012-02-21 04:51:24.000000000 +1100
@@ -70,15 +70,6 @@
         assertEquals("This is a comment", descriptor.getDescription(ExifDirectory.TAG_USER_COMMENT));
     }
 
-    public void testUserCommentDescription_AsciiHeaderExtendedAsciiEncoding() throws Exception
-    {
-        byte[] commentBytes = "ASCII\0\0This is a comment with extended characters זרו ֶ״ֵ".getBytes();
-        ExifDirectory directory = new ExifDirectory();
-        directory.setByteArray(ExifDirectory.TAG_USER_COMMENT, commentBytes);
-        ExifDescriptor descriptor = new ExifDescriptor(directory);
-        assertEquals("This is a comment with extended characters זרו ֶ״ֵ", descriptor.getDescription(ExifDirectory.TAG_USER_COMMENT));
-    }
-
     public void testUserCommentDescription_AsciiHeaderAsciiEncoding() throws Exception
     {
         byte[] commentBytes = "ASCII\0\0This is a comment".getBytes();
